main.js

```diff
@@ -1 +1,46 @@
 import customers from './customers.js'
+
+
+let person = [{
+    thumbNail: "https://randomuser.me/api/portraits/thumb/women/89.jpg",
+    name: "Pauline Hughes",
+    email: "pauline.hughes@example.com",
+    address: "3747 Elgin St, Roseville, IN 93281"
+}]
+
+
+function listCustomers(personArray) {
+    let customerList = document.querySelector('#customer-list');
+    console.log(customerList)
+
+    for (let customer of customers) {
```

Comment:
You'll want to loop over `personArray`, which is the name of the parameter inside the function. You specify `customers` when you call the function. `listCustomers(customers)`